.gitignore

```diff
@@ -25,3 +25,32 @@ _testmain.go
 
 vendor
 .cover
+
+# MacOS
+
+*.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
```

Comment:
This is the complete list of system-generated files. For a good coverage of systems I always use this list. It originates from : https://github.com/github/gitignore/blob/master/Global/macOS.gitignore